src/arrayMethodSort.js

```diff
@@ -5,7 +5,24 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compareDefault = (a, b) => Number(a.toString() > b.toString());
+
+    const compare = typeof compareFunction !== 'function'
+      ? compareDefault
+      : compareFunction;
```

Comment:
```suggestion
const compare = compareFunction || compareDefault;
```
operator || will return the first truthy value. looks more convenient to me)